MoonDawgArduino/Libraries/Movement/Movement.properties

```diff
@@ -0,0 +1,17 @@
+name=MovementMoonDawg
+
+version=0.8.0
+
+author=Cameron Schwartzberg
+
+maintainer=camhotwheels@gmail.com
+
+sentence=This contains all of the movement classes and functions for the MoonDawgArduinoCode.ino.
+
+paragraph=To be added later.

```

Comment:
should this be added

Reply:
?

Reply:
Yes it's part of the .properties file which is used when you zip a library for the Arduino IDE